### DISCUSSION TITLE
Help how to create a form field without relating it to a model column

### DISCUSSION THREAD

**Discussion by MarJose123** [filamentphp/filament#3514] [Help] [answered]
like for example I have a birthday fields and database column, and now I want to calculate the age and show it on view only, but I don't want it to save to the database, is there a way for this?

**Comment by mikrosmile** [comment]
Create either textinput or placeholder with the name you'd like ->make("age") 
Use stateRecordUsing method to calculate age based on $record->birthdate and function which will calculate your age. And show it only on table and view page of recourse with the method visibleOn("view")

**Comment by danharrin** [comment, marked as answer]
```php
DatePicker::make('dob')->visibleOn('edit'),
Placeholder::make('age')->content(fn (Model $record) => $record->dob->age)->visibleOn('view'),
```
